bento-compose/src/main/java/com/yelp/android/bento/compose/ComposeViewHolder.kt

```diff
@@ -23,9 +22,6 @@ abstract class ComposeViewHolder<P, T> : ComponentViewHolder<P, T>() {
 
     final override fun inflate(parent: ViewGroup): View {
         return ComposeView(parent.context).apply {
-            setViewCompositionStrategy(
-                ViewCompositionStrategy.DisposeOnViewTreeLifecycleDestroyed
-            )
             setContent {
                 state = remember { mutableStateOf(element) }
```

Comment:
It looks like in the medium blog post they use a different strategy to this:

```kotlin
    fun bind(input: String) {
        composeView.setContent {
            MdcTheme {
                Text(input)
            }
        }
    }
```
We should probably switch the implementation here to match. Thoughts?